data/error_policies/derive_diff_associated_call.cas

```diff
@@ -0,0 +1,18 @@
+virtual resource foo {
+	@associated_call
+	fn some_associated_call(domain source) {
+		allow(source, this, file, link);
+	}
+}
+
+virtual resource bar {
+	fn some_associated_call(domain source) {
+		allow(source, this, dir, add_name);
+	}
+}
+
+@derive([some_associated_call], strategy=union)
```

Comment:
Could we remove the strategy argument for now? It would be handy for other strategies but there is only `union` for now, so I think it doesn't make sense to keep the `strategy` argument.

Reply:
It's already been removed.  I just missed removing it in this error test.  Will push an update.